src/messages.rs

```diff
@@ -95,6 +95,7 @@ pub enum IncomingMessages {
     WshEventData = 105,
     HistoricalSchedule = 106,
     UserInfo = 107,
+    FamilyCodeEnd = 108,
```

Comment:
I don't see this code defined in IB source

Reply:
Will remove code.